app/models/ext_management_system.rb

```diff
@@ -20,11 +20,11 @@ def self.types
     concrete_subclasses.collect(&:ems_type)
   end
 
-  def self.supported_types
-    supported_subclasses.collect(&:ems_type)
+  def self.permitted_types
+    permitted_subclasses.collect(&:ems_type)
   end
 
-  def self.supported_subclasses
+  def self.permitted_subclasses
     concrete_subclasses.select(&:permitted?)
```

Comment:
:+1: `permitted_subclasses` makes much more sense

Reply:
It also aligns with the `.permitted?` method which was already there.